src/router/index.js

```diff
@@ -21,6 +24,21 @@ const routes = [
     name: 'Profile',
     // add component later
   },
+  {
+    path: '/UserResView',
+    name: 'UserResView',
+    component: UserResView,
+  },
+  {
+    path: '/UserReviewResView',
+    name: 'UserReviewResView',
+    component: UserReviewResView,
+  },
+  {
+    path: '/UserConfirmResView',
+    name: 'UserConfirmResView',
+    component: UserConfirmResView,
+  },
```

Comment:
In the router, only two views will exist with each component. Having a view for every component is redundant and unnecessary.